report_wkhtmltopdf_param/models/report.py

```diff
@@ -22,6 +22,17 @@ def _build_wkhtmltopdf_args(
         )
 
         for param in paperformat_id.custom_params:
+            try:
+                value_index = command_args.index(param.name)
+                if param.value:
+                    command_args[value_index + 1] = param.value
+                else:
+                    pass  # param.name is already in command_args
+                is_existing = True
+            except ValueError:
+                is_existing = False
```

Comment:
Wouldn't be cleaner to remove the is_existing and do the command_args.exentd and so on inside the except?

Reply:
Thanks @etobella for the suggestion. It has been updated.